packages/protocol/src/mappings/SavingsContract.ts

```diff
@@ -237,3 +237,9 @@ function updateTotalSavings(
     )
   }
 }
+
+export function handleFractionUpdated(event: FractionUpdated): void {
+  let savingsContractEntity = getOrCreateSavingsContract(event.address, null)
```

Comment:
The savings contract entity will always exist when this event is received, so you can just call `.save` without loading the entity, and the store will only write the fields changed here (leaving the others as-is)

```
let savingsContractEntity = new SavingsContractEntity(event.address.toHexString())
savingsContractEntity.fraction = event.params.fraction
savingsContractEntity.save()
```